tachyon/base/containers/circular_deque.h

```diff
@@ -1091,7 +1091,7 @@ class circular_deque {
   // Danger, the buffer_.capacity() is the "internal capacity" which is
   // capacity() + 1 since there is an extra item to indicate the end. Otherwise
   // being completely empty and completely full are indistinguishable (begin ==
-  // end). We could add a separate flag to avoid it, but that adds significant
+  // end). We could add a separate flag to avoid it, but that adds a significant
```

Comment:
Since "adds complexity" is the common term used in coding, I think "adds **a significant amount of** extra complexity" would be better.